src/core/dbus-cgroup.c

```diff
@@ -2319,6 +2319,117 @@ int bus_cgroup_set_property(
                 return 1;
         }
 
+        if (streq(name, "DevMemoryMax")) {
+                CGroupDeviceMemoryLimit *l;
+                char *region;
+                uint64_t size;
+
+                r = sd_bus_message_enter_container(message, 'a', "(st)");
+                if (r < 0)
+                        return r;
+
+                while ((r = sd_bus_message_read(message, "(st)", &region, &size)) > 0) {
+                        if (size < 1)
+                                return sd_bus_error_setf(
+                                                error,
+                                                SD_BUS_ERROR_INVALID_ARGS,
+                                                "Value specified in %s is out of range",
+                                                name);
+
+                        if (!UNIT_WRITE_FLAGS_NOOP(flags)) {
+                                l = NULL;
+
+                                LIST_FOREACH(dev_limits, cur, c->dev_mem_limits) {
+                                        if (!strcmp(cur->region, region)) {
```

Comment:
in our codebase, please use streq(), not strcmp(), unless you are interested in ordering rather than just equality